addon/components/sequential-render.js

```diff
@@ -20,6 +20,7 @@
       context=pageContext
       taskName='fetchPrimaryContent'
       fetchDataTask=fetchPrimaryContent
+      getData=executePromise
```

Comment:
Lets remove fetchDataTask from here and all other examples / tests (Except the direct tests on that API) as its going to be tagged for deprecation

Reply:
Similarly, lets also do the same for all the other deprecated keys

Reply:
Have addressed in latest commit